src/Compilers/VisualBasic/Portable/SymbolDisplay/SymbolDisplay.vb

```diff
@@ -105,7 +105,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             Dim builder = ArrayBuilder(Of SymbolDisplayPart).GetInstance()
             Dim visitor = New SymbolDisplayVisitor(builder, format, semanticModelOpt, positionOpt)
-            symbol.Accept(visitor)
+            visitor.VisitSymbol(symbol)
```

Comment:
why was this change made?

Reply:
This was because we do not accept visitors for pp symbols, but the FAR visitor visits pp symbols, and that specific visitor specially handles pp symbols around that lack of support for visitors from pp symbols. Therefore, propagating the visit responsibility to the visitor itself makes this work, while also respecting other symbols' visitor acceptance implementations.

Reply:
> This was because we do not accept visitors for pp symbols,

It seems like that is what should be fixed.

Reply:
Initially I overloaded the Accept methods in PreprocessingSymbols to support this symbol display, but then I was directed away from that, so I had to adjust the code here to make it work.

Reply:
i don't see any changes where you overloaded the accept methods.  I see changes where you make the Accept methods be block-bodied instead of expr-bodied.  You were asked to revert those unnecessary syntactic changes.

Reply:
Those were the first changes, and they changed the throw expression into a default visit for the visitor:
https://github.com/dotnet/roslyn/pull/66425/commits/1036dc5ce0739bcd95c1c189474f851e41f75058#diff-a39d8c0c96cd585796c892c4e33a3171b6e0e297bf8ad554a2923d5fb10cc71eR72

By implementing the accept visitor methods, like they were in the first commit, the symbol display does not need any changes.

Reply:
The code I commented on shows:

```
        void ISymbol.Accept(SymbolVisitor visitor)
+        {
+            throw new NotSupportedException();
+        }
```

That is what I responded to. An unnecessary style change

Reply:
This is what I see:
https://github.com/dotnet/roslyn/pull/66425/commits/1036dc5ce0739bcd95c1c189474f851e41f75058#r1089320057